lib/linux/background-managers/cinnamon.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const {commandExists, execFile, grep} = require('../util');
+
+exports.isAvailable = async () => {
+	if (!await commandExists('gsettings')) {
+		return false;
+	}
+
+	try {
```

Comment:
Here it's ok with try/catch as it's expected it might fail, but it should return `false` in the catch clause.